storage/innobase/buf/buf0buf.cc

```diff
@@ -1943,11 +1943,9 @@ inline void buf_pool_t::resize()
 	}
 #endif /* BTR_CUR_HASH_ADAPT */
 
-	if (!warning) {
-		buf_resize_status("Completed resizing buffer pool.");
-	} else {
+	if (warning) {
 		buf_resize_status("Resizing buffer pool failed");
-	}
+	} 
```

Comment:
try not to leave trailing spaces. Braces can be removed as the style is slowly changing.

Reply:
Good catch! Braces removed.

I noticed we have a `.clang-format` file under our project root directory. Is it OK to format the source file that is touched with those settings? I've got the `format-all` package in my Emacs, and have it hooked to `clang-format` program, so it's quite easy to do.

Reply:
Last I saw the clang-format couldn't reflect the current style. I still just copy the style that's nearby.